doc/source/user/basics.creation.rst

```diff
@@ -75,10 +75,10 @@ assign a new type that satisfies all of the array elements involved in
 the computation, here ``uint32`` and ``int32`` can both be represented in
 as ``int64``. 
 
-The default NumPy behavior is to create arrays in either 64-bit signed
-integers or double precision floating point numbers, ``int64`` and
-``float``, respectively. If you expect your arrays to be a certain type,
-then you need to specify the ``dtype`` while you create the array. 
+The default NumPy behavior is to create arrays in either 32 or 64-bit signed 
+integers (platform dependent) or double precision floating point numbers, 
```

Comment:
Not sure platform dependent totally captures it.  Might say "platform dependent and matches C int size".